projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -173,6 +178,39 @@ public function set_site_card_menu_class( array $menu ) {
 		return $menu;
 	}
 
+	/**
+	 * Returns the first available upsell nudge.
+	 *
+	 * @return array
+	 */
+	public function get_upsell_nudge() {
+		require_lib( 'jetpack-jitm/jitm-engine' );
+		$jitm_engine = new JITM\Engine();
+
+		$message_path = 'calypso:sites:sidebar_notice';
+		$current_user = wp_get_current_user();
+		$user_id      = $current_user->ID;
+		$user_roles   = implode( ',', $current_user->roles );
+		$query_string = array(
+			'message_path' => 'calypso:sites:sidebar_notice',
```

Comment:
Can we reuse `$message_path` here?
```suggestion
			'message_path' => $message_path,
```

Reply:
Right, 143e8f7